www/views/lights.html

```diff
@@ -26,328 +26,354 @@
 	</form>
 </div>
 <div id="dialog-addmanuallightdevice">
```

Comment:
Could you undo all formatting changes in the HTML ? Looks you replaced tabs with spaces, very hard to see what has changed
Please only changes....